### PR TITLE
Add a generic slideshow delegate

### PR DESCRIPTION
**Pull request by AIIX**
Adds a generic slideshow delegate which can be used to display cards on autoplay.
Example Usage: 
`
Mycroft.SlideshowDelegate {
   id: root
   model: sessionData.exampleModel  // model with slideshow data
   interval: 5000 // time to switch between slides
   running: true // can be set to false if one wants to swipe manually
   delegate: Kirigami.AbstractCard {
    ....
    }
} 
`

### CODE REVIEW

File: import/qml/SlideshowDelegate.qml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    Timer {
+        id: slideShowTimer
+        interval: 5000
+        running: false
+        repeat: true
+        onTriggered: {
+            var getCount = slideShowView.count
+            if(slideShowView.currentIndex !== getCount){
+                slideShowView.currentIndex = slideShowView.currentIndex+1;
+            }
```
Comment (notmart) [inline import/qml/SlideshowDelegate.qml]: } else {
in a single line

File: import/qml/SlideshowDelegate.qml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            else{
+                slideShowView.currentIndex = 0
+            }
+        }
+    }
+
+    contentItemAutoHeight: false
+    contentItem: ListView {
```
Comment (notmart) [inline import/qml/SlideshowDelegate.qml]: maybe a default delegate with a simple Image?

File: import/qml/SlideShow.qml [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+ *
+ */
+
+import QtQuick 2.4
+import QtQuick.Controls 2.2 as Controls
+import QtQuick.Layouts 1.3
+import org.kde.kirigami 2.5 as Kirigami
+import Mycroft 1.0 as Mycroft
+
+Flickable {
```
Comment (notmart) [inline import/qml/SlideShow.qml]: Item {

File: import/qml/SlideShow.qml [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    id: root
+    
+/**
```
[the remark this answers is not in the document]
Comment (AIIX) [reply import/qml/SlideShow.qml]: fixed in newer commit

**Pull request merged**